### DISCUSSION TITLE
CIDRAM + e107 as plugin

### DISCUSSION THREAD

**Discussion by rica-carv** [CIDRAM/CIDRAM#586] [General]
Hi to all.

Just asking, could i try to make a e107 plugin and try to implement CIDRAM in this CMS?

**Comment by Maikuolan** [comment]
Hi @rica-carv,

Certainly, you can do that. :-)

In general, anyone willing to put in the time and effort to create new plugins for any of the numerous CMS, frameworks, and other systems out there to help better integrate CIDRAM to those systems would be very welcome to do so.

I'm not familiar with e107, beyond knowing the name, and seeing it in passing or mentioned in online discussions, and I haven't used it before, so I've no idea what would be involved exactly in making a plugin for it, or how easy/difficult it would be to do so. If you're willing to try it though, I'd be happy for you to do so.

**Reply by rica-carv** [reply]
Hi @Maikuolan 
Thanks
I once had CIDRAM setup on a e107 setup, but it was a long time ago, so i'll give it a try again...
Thanks
